### PR TITLE
Manthan patel hw1

### PR DESCRIPTION
**Pull request by manthan2905**

### CODE REVIEW

**Review by iambillmccann — CHANGES_REQUESTED** [review]
You did not follow the assignment instructions. This is causing you to add a large number of files that do not belong in this code ... /EditPostPage, /NewPostPage, scaffold.css, etc. You need to revert your commits back to a point where there is only changes to Routes.js and /ManthanpatelPage.

File: web/src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'
 import FatalErrorPage from 'src/pages/FatalErrorPage'
 import Routes from 'src/Routes'
 
+import './scaffold.css'
```
Comment (iambillmccann) [inline web/src/App.js:9]: You should not make any changes to App.js for this assignment.

**Review by iambillmccann — CHANGES_REQUESTED** [review]
I am curious how you managed to get those extraneous files into your code. You still need to back out the changes made to App.js.

**Comment by manthan2905** [comment]
I have by mistake used the terminal command under read me in the main branch and it added all the posts and added those extra files post. Thank you for your output I didn't know it kept all the changes. It did work under redwood when I checked so I thought it was done. My fault

**Review by iambillmccann — APPROVED** [review]

**Pull request merged**